multimedia/v4l2tools/Makefile

```diff
@@ -31,8 +29,8 @@ define Package/v4l2tools
 endef
 
 define Package/v4l2tools/install
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/v4l2tools-$(PKG_VERSION) $(1)/usr/bin/v4l2tools
+  $(INSTALL_DIR) $(1)/usr/bin
```

Comment:
Actually, the previous solution was correct. Install section needs to have as an indentation tab, but description two spaces.